src/api/batch/farmlands.md

```diff
@@ -0,0 +1,316 @@
+| externalId | String             | Id used for keeping imported [Patron Code][] details up to date                                                       |
+| status     | String             | Current state of the Card. Valid values are "active", "inactive" and "archived"                                       |
+| expiry     | {% dt Timestamp %} | Payments will be accepted until this time. When expressed as `YYYY-MM-DD` the Card will be valid up to 23:59:59 NZST. |
+
+{% h4 Optional Fields %}
+
+|      Field      |  Type  |                              Description                              |
+| :-------------- | :----- | :-------------------------------------------------------------------- |
+| barcode         | String | 9 digit field to display in barcode format                            |
+| farmlandsStatus | String | Private field used in all Farmlands payment transaction notifications |
+
+
+<a name="example">
+## Examples
+
+Full Account, Contact and Card Update
+{% json %}
+{
+  externalId: "b5fde0e0-357c-4fda-a90f-fd857f2be999",
+  accountNumber: "830578479",
+  availableBalance: "12000",
+  contacts: [
+    {
+      externalId: "6e496c2a-1dae-4036-847d-c53bf6c6d410",
+      name: "Road Runner",
+      mobile: "+64221102598",
+      email: "road@runner.net",
+      primary: true,
+      cards: [
+        {
+          externalId: "65e701c3-6973-4322-8fa6-4560a489417f",
+          barcode: "458028560",
+          status: "active",
+          expiry: "2023-06-01T22:32:56.631Z",
+        },
+        {
+          externalId: "69d64d80-f9bd-4057-bc5b-1c55685d995b",
+          barcode: "635570865",
+          status: "inactive",
+          farmlandsStatus: "Suspended by customer",
+          expiry: "2022-04-01T22:32:56.631Z",
+        }
+      ]
+    },
+    {
+      externalId: "6e4813e6-7a18-47ea-b92e-add36c8815ca",
+      name: "Yosemite Sam",
+      mobile: "+64220002598",
+      email: "yosemite@runner.net",
+      primary: false,
+      cards: [
+        {
+          externalId: "74e4f94c-8316-42e7-9aa1-eb1539528894",
+          barcode: "137628567",
+          status: "active",
+          expiry: "2025-06-01",
+        }
+      ]
+    },
+    {
+      externalId: "82bdb041-ea79-448c-816f-77af8b6750b2",
+      name: "Wile E. Coyote",
+      mobile: "+64221102598",
+      email: "while@e-cyote.net",
+      primary: false,
+      cards: [
+        {
+          externalId: "62904b86-b4cc-45a9-b3c9-287a00ae9ef5",
+          barcode: "722798445",
+          status: "active",
+          expiry: "2025-05-07",
+        }
+      ]
+    }
+  ]
+}
+{% endjson %}
+
+Balance Update and Deleting Contacts
+{% json %}
+{
+  externalId: "9b2ec6d1-c83b-496a-8e52-2989f23d9076",
+  accountNumber: "012345678",
+  availableBalance: "1000",
+  contacts: [],
+}
+{% endjson %}
+
+Balance Update
+{% json %}
+{
+  externalId: "d0d7e14d-4ce5-4f42-8a4c-d604a9609f66",
+  accountNumber: "012345678",
+  availableBalance: "1000",
+}
+{% endjson %}
+
+Contact Update
```

Comment:
Just removing the title from this isn't enough. I expect to see a sentence explaining the example.

```suggestion
Update the contact list for an account. Use this where you want to exclude the balance.
```